crates/kitsune_p2p/kitsune_p2p/src/config.rs

```diff
@@ -46,6 +48,10 @@ impl Default for KitsuneP2pConfig {
             bootstrap_service: None,
             tuning_params: KitsuneP2pTuningParams::default(),
             network_type: NetworkType::QuicBootstrap,
+            #[cfg(feature = "sharded")]
+            resize_arcs: true,
+            #[cfg(not(feature = "sharded"))]
+            resize_arcs: false,
```

Comment:
have we not done the work to switch from feature flag to tuning param yet?

I thought the intention was to not have to recompile to alter this behavior.

Reply:
This is a bit of a placeholder. This "feature" isn't even hooked up anywhere else, kind of just rolling with what we had before. Note that this is just changing the default value of the config. I could hook this up more properly or leave a comment about it, just not sure how far to go with it.

Reply:
for sure, I'm fine with the work to be done in a different effort, just don't want to forget about it : )

Reply:
Probably want to remove this now that we do have a PR for this?